addons/event/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,26 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def fill_event_mail_notification_type(cr):
```

Comment:
Delete this method, it will do nothing. During the update of the module (between the pre-migration and the post-migration), both notification_type columns will be created and automatically filled by their default value.

Reply:
does always odoo fill default values of new fields?

Reply:
yes. if a field has default, and the field is new, then odoo fill the new column with the default.